### DISCUSSION TITLE
Clone the repo as well if it is Github?

### DISCUSSION THREAD

**Discussion by gerroon** [ArchiveBox/ArchiveBox#1302] [Q&A] [answered]
Hi

Is it possible to archive the git repo if the user is archiving a Github page?

**Comment by pirate** [comment, marked as answer]
This is already built-in by default, `archivebox config --set SAVE_GIT=True` clones the source code repo on any Github/Gitlab/Bitbucket dirs into the snapshot directory.

- https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#save_git
- https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#GIT_DOMAINS
- https://github.com/ArchiveBox/ArchiveBox/blob/dev/archivebox/extractors/git.py#L59
- https://demo.archivebox.io/archive/1694477188.931182/index.html#git

<img width="1501" alt="image" src="https://github.com/ArchiveBox/ArchiveBox/assets/511499/50366a61-45c7-4704-97e1-0d3b243c1ef0">

**Reply by gerroon** [reply]
Awesome, thanks
